### PR TITLE
FAT-5151-C378883

### PR DESCRIPTION
**Pull request by TetianaParanich**
***
https://issues.folio.org/browse/FAT-5151
***
https://foliotest.testrail.io/index.php?/cases/view/378883
***
https://jenkins-aws.indexdata.com/job/Testing/job/Cypress_tests/2740/allure/
***

### CODE REVIEW

File: cypress/e2e/data-import/importing-marc-bib-files/import-additional-files-without-handling-after-stopping-running-job-and-deleting-it.cy.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    { tags: [TestTypes.criticalPath, DevTeams.folijet] }, () => {
+      cy.visit(TopMenu.dataImportPath);
+      // TODO delete function after fix https://issues.folio.org/browse/MODDATAIMP-691
+      DataImport.verifyUploadState();
+      DataImport.uploadFile('marcFileForC356824.mrc', bigFileName);
+      cy.wait(5000);
+      JobProfiles.searchJobProfileForImport(jobProfileToRun);
+      JobProfiles.runImportFile();
+      Logs.checkFileIsRunning(bigFileName);
+      cy.wait(5000);
```
Comment (IhorBohdan) [inline cypress/e2e/data-import/importing-marc-bib-files/import-additional-files-without-handling-after-stopping-running-job-and-deleting-it.cy.js:48]: remove waiter to method or explain why it needs here

File: cypress/support/fragments/data_import/dataImport.js [hunk 2]
```diff
@@ -341,5 +341,23 @@ export default {
...
+  verifyCancelImportJobModal:() => {
+    cy.expect([
+      Modal('Cancel import job?').exists(),
+      Modal('Cancel import job?').find(Button('Yes, cancel import job')).exists(),
+      Modal('Cancel import job?').find(Button('No, do not cancel import')).exists()
+    ]);
+  },
+
+  cancelImportJob:() => {
+    cy.do(Modal('Cancel import job?').find(Button('Yes, cancel import job')).click());
```
Comment (IhorBohdan) [inline cypress/support/fragments/data_import/dataImport.js:361]: Move button also const

**Review by IhorBohdan — APPROVED** [review]

**Review by nayimovag — APPROVED** [review]

File: cypress/e2e/data-import/importing-marc-bib-files/import-additional-files-without-handling-after-stopping-running-job-and-deleting-it.cy.js (hunk 1 above)
Comment (TetianaParanich) [reply cypress/e2e/data-import/importing-marc-bib-files/import-additional-files-without-handling-after-stopping-running-job-and-deleting-it.cy.js:48]: Done

File: cypress/support/fragments/data_import/dataImport.js [hunk 3]
```diff
@@ -341,5 +341,23 @@ export default {
...
+
+  deleteImportJob:(fileName) => {
+    cy.get('div[class^="listContainer-"]').contains('li[class^="job-"]', fileName).then(elem => {
+      elem.parent()[0].querySelector('button[icon="trash"]').click();
+    });
+  },
+
+  verifyCancelImportJobModal:() => {
+    cy.expect([
+      Modal('Cancel import job?').exists(),
```
[the remark this answers is not in the document]
Comment (TetianaParanich) [reply cypress/support/fragments/data_import/dataImport.js:354]: Done

File: cypress/support/fragments/data_import/dataImport.js (hunk 2 above)
Comment (TetianaParanich) [reply cypress/support/fragments/data_import/dataImport.js:361]: Done

**Pull request merged**